gyp/pylib/gyp/MSVSNew.py

```diff
@@ -20,6 +20,11 @@
   import md5
   _new_md5 = md5.new
 
+try:
```

Comment:
Could you move these to __init__.py, then import them (`cmp`, `type_list`, `basestring`, etc..)